src/modules/mavlink/mavlink_receiver.h

```diff
@@ -293,6 +300,13 @@ class MavlinkReceiver : public ModuleParams
 
 	hrt_abstime			_last_utm_global_pos_com{0};
 
+	static constexpr unsigned MAX_TUNE_LEN {248};
+
+	// Allocated if needed.
+	Tunes *_tunes {nullptr};
```

Comment:
What about moving these into a separate struct/class? Then you'd only have one allocation.

Reply:
@bkueng done inc8c690d4da86122ac5d58b11eca8fa659d2110b8.